.github/workflows/release.yml

```diff
@@ -90,7 +90,7 @@ jobs:
         uses: rtCamp/action-slack-notify@v2
         env:
           SLACK_WEBHOOK: ${{ secrets.SLACK_WEBHOOK }}
-          SLACK_CHANNEL: wonder-blocks-eng
```

Comment:
**question:** I wonder if we should move these to configurable environment vars so that we don't have to ship code changes just to udpate them?

Reply:
That's a great idea!  I'll do that.

Reply:
I ended up having to add it a secret.